content/faq.md

```diff
@@ -18,3 +18,23 @@ menu:
 Anyone that wants to integrate services. Syndesis includes a swish UI that enables the user to design integration flows and manage them from their browser.
 
 No coding required... Unless you really want to and then Syndesis allows you to dive into the code, develop your own connectors (if one doesn't already exist), or hack on the integration definition directly.
+
+## How to read logs?
+Logs are currently visible only at infrastructure level, not via the web ui.
+You can get a list of the pods running in Syndesis with the command: `oc get pods -n myproject`.  To read logs from command line, you can then invoke: `oc logs <POD_NAME>`
+
+## How to read the configuration files that are generated by the UI tool
+If you are a developer and you want to check the configuration files corresponding to the Integration that you have designed in the web ui, you can access them with the following commands:
+- `oc cp myproject/<POD_NAME>:/deployments/project-0.1-SNAPSHOT.jar prj.jar`
```

Comment:
You might want to mention that you can get the whole src by copying over `/tmp/src` from the integration pod as this is the directory that is used by the S2I build.